packages/dashboard/src/components/composed/Sidebar/Middle/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Navbar, Badge } from "@mantine/core";
+import { Zap, Archive, Aperture } from "react-feather";
+import NavButton from "src/components/composed/Sidebar/Middle/NavButton";
+
+function Middle(): JSX.Element {
+  const featherIconProps = { size: 18 };
+
+  const comingSoonBadge = (
+    <Badge
+      variant="gradient"
+      gradient={{ from: "truffle-teal", to: "pink" }}
+      sx={{ opacity: 0.4 }}
+    >
+      coming soon
+    </Badge>
+  );
+
+  return (
+    <Navbar.Section grow py="sm">
+      <NavButton
+        label="Transactions / Calls"
```

Comment:
Calls won't actually prompt the user, since they don't need a signature... maybe better to call this "Signature Requests" or somesuch

Reply:
It would also probably be a nice affordance to include a badge with the number of outstanding requests

Reply:
I wasn't sure on how to label this, because a given request may not be a transaction request (e.g. `eth_signTransaction`, `personal_sign`, etc.), but I erroneously put "calls". "Signature Requests" is much better! Commit [here](https://github.com/trufflesuite/truffle/pull/5241/commits/9a8fb1c2d72ebfd00eeef6edb68c1a978d6a439d).

I've also added a dot to indicate the number of outstanding requests (capping at 99, >99 shows "99+"). Commit [here](https://github.com/trufflesuite/truffle/pull/5241/commits/7996f8a3195ca6dac3a9f109e313c7a06a6bf892).